### DISCUSSION TITLE
Second order drug - protein reaction

### DISCUSSION THREAD

**Discussion by zt1947** [Open-Systems-Pharmacology/Forum#1710] [Questions & Problems]
How do you include a second order reaction (describing drug -  plasma protein interaction)  in a PK Sim model?

**Comment by wilbertdew** [comment]
Hi @zt1947, any protein binding partner that you create in PK-Sim will use a second order reaction. However, if you want to do that for a circulating protein like Albumin, you can either do this via a protein expression in vasculature (but you wont have distribution to tissues of that protein) or you can make Albumin a molecule similar to a drug molecule and implement the binding reactions manually in MoBi.

**Comment by zt1947** [comment]
Thanks for the info!

בתאריך יום ו׳, 1 במרץ 2024, 10:57, מאת wilbertdew ***@***.***
